SCClassLibrary/Common/GUI/PlusGUI/Control/scopeResponse.sc

```diff
@@ -27,6 +27,7 @@ Slew.scopeResponse
 		server = server ? Server.default;
 		if (server.serverRunning.not) {
 			Error("Function-scopeResponse: server not running").throw
+			"server '%' not running".format(server.name).warn;
```

Comment:
Similarly, no need to add a warning here,

Reply:
Done https://github.com/supercollider/supercollider/pull/6796/commits/6402765df63066721da49fba102a46a0d6571baa!